core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -31,6 +31,8 @@ var ExplorationEditorPage =
 var ExplorationPlayerPage =
   require('../protractor_utils/ExplorationPlayerPage.js');
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
+const { element } = require('protractor');
+const { expect } = require('globalthis/implementation');
```

Comment:
Why are these needed?

Reply:
I guess these were added by mistake. Sorry for not checking these before asking for a review.